macros/GroupData.json

```diff
@@ -1822,10 +1822,33 @@
         },
         "WebRTC": {
             "overview":   [ "WebRTC API" ],
+            "guides":     [ { "url":   "/en-US/docs/Web/API/WebRTC_API/Protocols",
+                              "title": "Introduction to WebRTC protocols" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Connectivity",
+                              "title": "WebRTC connectivity" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Session_lifetime",
+                              "title": "Lifetime of a WebRTC session" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Signaling_and_video_calling",
+                              "title": "Signaling and video calling" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Using_data_channels",
+                              "title": "Using WebRTC data channels" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Using_DTMF",
+                              "title": "Using DTMF with WebRTC" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/adapter.js",
+                              "title": "Improving compatibility using WebRTC adapter.js" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Taking_still_photos",
+                              "title": "Taking still photos with WebRTC" },
+                            { "url":   "/en-US/docs/Web/API/WebRTC_API/Simple_RTCDataChannel_sample",
+                              "title": "A simple RTCDataChannel example" } ],
+
             "interfaces": [ "MediaStreamEvent",
                             "RTCConfiguration",
                             "RTCDataChannel",
                             "RTCDataChannelEvent",
+                            "RTCDTMFSender",
+                            "RTCDTMFToneChangeEvent",
+                            "RTCIceCandidate",
+                            "RTCIceTransport",
```

Comment:
These are all absolutely implemented. The DTMF stuff is in the process of being documented right now (indeed, the reference pages are done already aside from examples which are inbound). RTCIceCandidate is a core part of WebRTC and has been around since the very beginning. RTCIceTransport is also a core part of WebRTC and if it's not implemented, I'm fairly surprised, but I will double-check on it.